bots/tests-scan

```diff
@@ -44,9 +44,9 @@ DEFAULT_VERIFY = {
 
 # Non-public images used for testing
 REDHAT_VERIFY = {
-    # FIXME: We don't have a rhel-7-7 image yet, so test rhel-7.7 branch on 7.6 for now
-    "verify/rhel-7-6": [ 'rhel-7.6', 'rhel-7.7' ],
+    "verify/rhel-7-6": [ 'rhel-7.6' ],
     "verify/rhel-7-6-distropkg": [ 'master' ],
+    "verify/rhel-7-7": [ 'rhel-7.7' ],
```

Comment:
The bots won't recognize this yet. You first need a quick PR like https://github.com/cockpit-project/cockpit/pull/11360 to declare the image, then *this* PR would add the rhel-7.7 image to the test.

Reply:
#11375